### ISSUE TITLE
`enableUnhandledRejection` silently swallows errors instead of exiting in nodejs

### ISSUE THREAD

**Issue by thatsmydoing** [honeybadger-io/honeybadger-js#946]
## What are the steps to reproduce this issue?
```
const Honeybadger = require('@honeybadger-io/js');

Honeybadger.configure({
  ...
});

async function main() {
  throw new Error('AHHHH');
}

main();
```

## What happens?
The program exits successfully.

## What were you expecting to happen?
The program should exit with an error because of an unhandled promise rejection which is the standard behavior in nodejs.

## What versions are you using?
**Operating System:** Linux
**Package Name:** @honeybadger-io/js
**Package Version:** 4.7.1
**Browser Version:** n/a

**Comment by subzero10** [comment]
@thatsmydoing This should be fixed with `v4.7.4`!

**Comment by thatsmydoing** [comment]
Thanks! I can confirm it's fixed.
